certora/AllocatorOracle.spec

```diff
@@ -0,0 +1,23 @@
+// AllocatorOracle.spec
```

Comment:
Side note: for a more general oracle I'd suggest adding a property to check that `peek` and `read` always return equivalent values, but here there's no point here because that property follows from the two existing rules.